src/lamplib/tasks/run_tests.py

```diff
@@ -110,7 +110,11 @@ def resmoke_test(env, suites, mongo_dir, is_cnats):
         # Default download location for MongoDB binaries.
         env["PATH"] += ":" + os.path.join(mongo_dir, "bin") + ":" + mongo_dir
 
-    evg_venv_dir = os.path.join(os.environ['SCRIPTS_DIR'], "venv")
+    evg_venv_dir = os.path.join(os.environ['LAMP_VENV_DIR'], "venv")
+
+    if not os.path.exists(evg_venv_dir,):
+        raise ValueError("The venv directory is required for resmoke and does not exist, "
+                         "please ensure you're running Genny in a virtualenv")
```

Comment:
I would change this error message to "please ensure you're not running Genny with the -g/--run-global flag." However, now I"m afraid we're basically just undoing the work from this ticket. Or are they completely separate worlds?